sys/shell/commands/sc_sntp.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2016 Luminița Lăzărescu <cluminita.lazarescu@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_shell_commands
+ * @{
+ *
+ * @file
+ * @brief       Prints the real time offset from the system time
+ *
+ * @author      Luminița Lăzărescu <cluminita.lazarescu@gmail.com>
+ */
+
+#include "net/sntp.h"
+#include "net/af.h"
+#include "net/ipv6/addr.h"
+
+int _ntpdate(int argc, char **argv)
+{
+    if (argc < 2) {
+        printf("Usage: %s <server addr>\n", argv[0]);
+	return 1;
+    }
+    sock_udp_ep_t server = {.port=123, .family=AF_INET6};
```

Comment:
Style: please surround operators with spaces and also add spaces around the whole initializer:

```C
    sock_udp_ep_t server = { .port = 123, .family = AF_INET6 };
```